### DISCUSSION TITLE
Remove all Budibase branding and use your own

### DISCUSSION THREAD

**Discussion by RedBlob** [Budibase/budibase#10530] [Help] [answered]
Hello,
Why on the Premium plan ( cloud hosted ) I can edit this feature but is not operative ?
Thank You.
Best Regards

**Comment by josh99smith** [comment]
Its a paid feature now, for self hosting  (which is a bit odd IMO) and cloud hosting.

**Reply by RedBlob** [reply]
Hello josh99smith,
First of all thank You for your help.

But we have a paid cloud plan , not a free plan ....... we have a Premium account on BB.

**Comment by RedBlob** [comment]
I have found this doc 
https://docs.budibase.com/docs/branding

This feature need a Cloud installations with at least a Pro license

What is the Pro plan ?
Now there is only Premium or Business or Enterprise versione.

We have a Premium plan and this feature is available in the backend panel ...... but don't work on front-end.

**Comment by RedBlob** [comment]
No suggestion ?

**Comment by RedBlob** [comment]
No suggestion ?

**Comment by shogunpurple** [comment]
@RedBlob can you please let us know your tenant ID in cloud so we can investigate? `<tenantid>.budibase.app`

**Comment by RedBlob** [comment]
Hello Shogunpurple, I have sent the tenantid by mail . 
I can't publish it on the web in this time ( company policy ).

**Comment by RedBlob** [comment]
Hello shogunpurple,
Are you sure this option in the Premium plan is working correctly ?
The login page of our account is with BB logo , not with a custom logo
![Immagine 2023-05-16 211807](https://github.com/Budibase/budibase/assets/70526244/bcaa8bcb-7be4-4602-831c-fc3459495576)

Thank You
Best Regards

**Reply by shogunpurple** [reply]
Hi @RedBlob - replied over email, but the login page branding is not available in cloud premium, only self host. You can however customise all the other branding elements behind the login.

**Comment by RedBlob** [comment]
Hello @shogunpurple ,
Thank you for yuor replay.
But sorry I don't understand the logic behind this choice.
I have a major project for a company with its own brand and this project involves 50-60-70 users.
If I activate the Cloud Premium plan on your server the cost will be about $400-500 per month and about $4000-5000 per year.
And even with this amount to budget I will have to tell my client that the login must be done on a page where the client's brand will not appear ?
In addition it also appears on the right hand side BB advertising referring to other companies or projects of institutional part as Bulgarian Gov ..... in short you are paying for a service to have third party sponsorship.

I don't think this is a good strategy.

**Comment by Rory-Powell** [comment]
Hi @RedBlob 

To access your custom branding at login time you should visit `<tenantId>.budibase.app`. This is a separate page specific to your tenant, rather than using the main `account.budibase.app` login which will be the same for every budibase cloud user. 

If you're not using SSO however, your own tenant login page will auto redirect to the account portal login. However there is a workaround, if you:
- Create an OIDC configuration (any values will do)
- Disable the integration
- Save
- Refresh the page
- Visit `<tenantId>.budibase.app`

You should now see your custom branding appear and not be redirected to the account portal.

**Comment by RedBlob** [comment]
Should this appear by enabling an OIDC authentication ?
<img width="1229" alt="Immagine 2023-05-17 123033" src="https://github.com/Budibase/budibase/assets/70526244/4b68d2a1-bfd9-47ed-a4f2-927295ecd9e8">
It just disappeared the BB logo but everything else is the same.
Could it be that there is no option to have the classic custom login with a logo and two text boxes for UserID and Password ? 
Similar to :
<img width="178" alt="Immagine 2023-05-17 123611" src="https://github.com/Budibase/budibase/assets/70526244/dd221735-fa7b-41c1-8c4b-d3f09295130b">

**Reply by Rory-Powell** [reply]
You really should be able to configure this in the cloud (like we can in self host). 
We'll push out an update that will give you more control over the per tenant login page in the cloud.

**Comment by RedBlob** [comment]
@Rory-Powell , Thank You very much for your help !
Waiting your update, Best regards

**Reply by RedBlob** [reply]
Hello @Rory-Powell ,
Do you have a roadmap for this development ? 
When will be pushed out ?
Thank You.
Best Regards

**Reply by Rory-Powell** [reply, marked as answer]
Hi @RedBlob, this has been scheduled for our next iteration and will go out in the June release

**Reply by RedBlob** [reply]
Hi @Rory-Powell 
Thank You so much for your feedback.
Best Regards.

**Comment by RedBlob** [comment]
Hello @Rory-Powell ,
There are news about this development ?
When do you think will be published ? In this new release or after ?
Thank You.
Best Regards

**Reply by Rory-Powell** [reply]
Hi @RedBlob, in our release cycle the monthly release name refers to the previous month. i.e. the May release went out at the beginning of June and the June release will go out at the start of July.

**Comment by RedBlob** [comment]
Hi @Rory-Powell ,
Thank You for feedback.
Best Regards

**Comment by RedBlob** [comment]
Hello @Rory-Powell ,
Are you sure this feature is present in this release ?
I have checked and from my side nothing is changed.
The login page is always the same ......
Thank You.
Best Regards

**Reply by Rory-Powell** [reply]
Hi @RedBlob . This update is present in the June release (2.8.1+)
The change was merged in: https://github.com/Budibase/budibase/pull/11032

You may need to visit your branding settings to verify the correct overrides have been put in place?
